### ISSUE TITLE
Is this project still alive?

### ISSUE THREAD

**Issue by apobbati** [stormpath/stormpath-scala#4]

**Comment by mrioan** [comment]
Hello @apobbati. We created this project as a proof of concept. We never officially released it. It was working though. You can find the code in the dev branch. Also, we integrated it with a sample Play app here: https://github.com/stormpath/stormpath-play-sample/tree/activator.

The latest versions of Play and Scala seem to have broken these two projects. We do not have any plans to update them in the near future either.

In summary, I would not call these projects dead but hibernating...
